F.R.O.S.T/app/src/main/java/t/s/o/r/f/frost/MainActivity.java

```diff
@@ -539,6 +542,11 @@ static void handleInput(){
                     displayTemp(value);
                     break;
                 case 'f': //Flame sensor input
+                    if(s.charAt(3) == '1'){
```

Comment:
This entire if statement just be, but It's just a suggestion.
fireImage.setVisibility(s.charAt(3) == '1' ? View.VISIBLE : View.INVISIBLE);